coolkit/dependencies/dependency.py

```diff
@@ -55,12 +43,28 @@ def is_installed(soft):
     return False
 
 
+def get_available_package_manager(dependency_map):
+    package_managers = set()
+    for rules in dependency_map.values():
+        for key in rules.keys():
+            package_managers.add(key)
+
```

Comment:
return package_managers.

Reply:
keep it like your are returning supported package managers. not any one of them.

Reply:
and make changes accrodingly in `install_dependencies`